core/src/main/java/org/apache/calcite/config/CalciteConnectionConfig.java

```diff
@@ -78,4 +78,6 @@
   boolean typeCoercion();
   /** @see CalciteConnectionProperty#LENIENT_OPERATOR_LOOKUP */
   boolean lenientOperatorLookup();
+  /** @see CalciteConnectionProperty#VOLCANO_TOPDOWN_OPT */
+  boolean enableVolcanoTopDownOptimization();
```

Comment:
Let's use `boolean topDownOpt()` to keep consistent with other method signatures.

Reply:
Done